modules/http/scanner.go

```diff
@@ -245,6 +283,11 @@ func (scan *scan) Grab() *zgrab2.ScanError {
 	}
 	// TODO: Headers from input?
 	request.Header.Set("Accept", "*/*")
+	if scan.scanner.customHeaders != nil {
```

Comment:
If custom headers are provided, should that list be the *only* headers that are sent? The way you have it implemented is probably the most intuitive, but at the cost of flexibility -- e.g. as it stands you can change the Accept header, but you cannot remove it.

Reply:
Good point, I'll fix it to match your recommendation.

Reply:
> If custom headers are provided, should that list be the _only_ headers that are sent? The way you have it implemented is probably the most intuitive, but at the cost of flexibility -- e.g. as it stands you can change the Accept header, but you cannot remove it.

I just noticed that if you separate this into two cases- one for when the user is specifying custom headers, one for where they're not- and have the custom header case *not* set the `Accept` header here- it just gets set later on in the HTTP library to a default value of `gzip`, which is probably not what you want

The code for that is  [here](https://github.com/zmap/zgrab2/blob/master/lib/http/transport.go#L1869) 

So maybe it is best to always assign `*/*` to `Accept` by default, allow the custom headers to override the value, and then if you want it to be removable, use a `--no-accept-header` flag? The only other option is to modify code in the HTTP library linked above. After noticing this, I'm inclined to leave it as is, but am curious what you think

Reply:
> So maybe it is best to always assign */* to Accept by default, allow the custom headers to override the value, and then if you want it to be removable, use a --no-accept-header flag?

It sounds like this is the simplest, if I understand correctly. If so, that sounds like a good option to me.